.github/workflows/builds.hex.pm.yml

```diff
@@ -107,6 +104,22 @@ jobs:
             aws s3 cp elixir-otp-${{ matrix.otp }}.zip "s3://${{ env.AWS_S3_BUCKET }}/builds/elixir/${{github.ref_name}}.zip" --cache-control "public,max-age=3600" --metadata "{\"surrogate-key\":\"builds builds/elixir builds/elixir/${{github.ref_name}}\",\"surrogate-control\":\"public,max-age=604800\"}"
             purge builds/elixir/${{github.ref_name}}
           fi
+  update_builds_txt:
+    needs: release_pre_built
+    concurrency: builds_txt
+    strategy:
+      matrix:
```

Comment:
Unfortunately we have to duplicate the matrix. I didn't find a good way of sharing it between jobs.

Reply:
Gah, this is tricky, very easy to update one and forget the other. :(

Reply:
Maybe better to update RELEASE.md with instructions to wait for action completion?